src/pages/beneficiaries/index.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-alert */
+/* eslint-disable no-underscore-dangle */
+import type { GetServerSideProps } from 'next';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { getBeneficiaries } from '../../apiHandlers/beneficiary';
+import { setState } from '../../store/stateSlice';
+
+export interface Beneficiary {
+    id: number;
+    name: string;
+    age: number;
+    gender: string;
+    occupation: string;
+    district: string;
+    thana: string;
+}
+
+export const getServerSideProps: GetServerSideProps = async () => {
+    const { success, beneficiaries } = await getBeneficiaries();
+
+    if (!success) {
+        return {
+            redirect: {
+                destination: '/',
+                permanent: false,
+            },
+        };
+    }
+
+    return {
+        props: { beneficiaries },
+    };
+};
+
+const Beneficiaries = ({ beneficiaries }: { beneficiaries: Beneficiary[] }) => {
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(setState({ title: 'Beneficiaries' }));
+    }, [dispatch]);
+
+    const tableHeader = ['name', 'age', 'gender', 'occupation', 'district', 'thana'];
```

Comment:
You have an array of table headers. And here you are rendering cell values using name. In long run, it might go out of sync, if you change the order of the elements in the table headers array.
Instead I will suggest you to use the following structure,
`const tableHeaders = [{ name: 'Age', key: 'age' }];`
Later `tableHeader.map(({ name }) => {}`
and in cells `beneficiary[header.key]`